File: string/58_length_of_last_word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        res = 0
        s = s.strip()
        n = len(s)
        while n:
            if s[n-1] != ' ':
                res += 1
            else:
                break
            n -= 1
        return res
        

File: aez/58_length_of_last_word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int res = 0;
        boolean flag = true;

        for (int i = s.length() - 1; i >= 0; i--) {
            if (flag && s.charAt(i) == ' ') {
                continue;
            } else if (s.charAt(i) == ' ') {
                break;
            } else {
                res++;
                flag = false;
            }
        }
        return res;
    }
}